examples/grafana/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM centos:7
```

Comment:
Is there a reason this needs to be in origin?  We're unlikely to build it, so it would be better as a standalone repo.

Reply:
Hmm, ok, will move it.

Reply:
Specifically the Dockerfile?  Fine to move.  @mrsiano you can put it in https://github.com/openshift/svt/tree/master/dockerfiles

Reply:
https://github.com/openshift/svt/tree/master/dockerfiles/grafana-ocp